test/onnx/softmaxcrossentropyloss_gold_data_check.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from onnx.reference import ReferenceEvaluator
+
+#X = np.array([[1.0,2.0,3.,4.], [2.,4.,5.,7.], [11.,13.,23.,17.], [2.,1.,31.,37.]], dtype=float)
+#X = np.array([[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]],
+#             dtype=float)
+#X = np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]],
+#             dtype=float)
+
+#X = np.array([[1, 1, 1, 1], [1, 1, 1, 1],[1, 1, 1, 1]], dtype=float)
+#X = np.array([[1, 1, 1], [1, 1, 1],[1, 1, 1], [1, 1, 1]], dtype=float)
+#X = np.array([[1, 1, 1, 1], [2, 2, 2, 2],[3, 3, 3, 3]], dtype=float)
+
+#X = np.array([[0, 0, 0, 0], [0 ,0 ,0 ,0], [0, 0, 0, 0], [0 ,0 ,0 ,0]], dtype=float)
+#X = np.array([[[[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]],
+#               [[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]]],
+#              [[[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]],
+#               [[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]]],
+#              [[[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]],
+#               [[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]]],
+#              [[[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]],
+#               [[1., 1.], [1., 1.]], [[1., 1.], [1., 1.]]]],
+#             dtype=float)
+
+X = np.array([[[[0., 0.], [0., 0.]], [[0., 0.], [0., 0.]],
```

Comment:
Pardon me, I don't understand the value in setting X to all zeros. And then weights to all 1's.

Reply:
Tests a known value, see later comment on the zeroes tests I did